lib/web/JsonpChunkLoadingRuntimeModule.js

```diff
@@ -69,7 +69,7 @@ class JsonpChunkLoadingRuntimeModule extends RuntimeModule {
 		if (options && options.baseUri) {
 			return `${RuntimeGlobals.baseURI} = ${JSON.stringify(options.baseUri)};`;
 		}
-		return `${RuntimeGlobals.baseURI} = document.baseURI || self.location.href;`;
+		return `${RuntimeGlobals.baseURI} = (document && document.baseURI) || self.location.href;`;
```

Comment:
Thanks for the PR! However this still fails

<img width="819" height="429" alt="Image" src="https://github.com/user-attachments/assets/09034ac7-f903-470a-aaf8-852089707b53" />

The correct check would have to use `typeof document` or `globalThis.document?.baseURI`

Reply:
PR welcome